ui/src/app/pages/profile/profile-registration-step-status.tsx

```diff
@@ -57,10 +59,11 @@ const ProfileRegistrationStepStatus: React.FunctionComponent<Props> =
       children,
       content
     } = props;
+    const titleId = useId();
```

Comment:
I was trying some things out to prevent the need for a data-test-id, but it didn't work out.
But I didn't want to lose the work done to add aria information, so I kept it in.